assets/src/application/ui/input/Button/ButtonGroup/style.scss

```diff
@@ -0,0 +1,20 @@
+.ee-btn-group {
+	.ee-btn {
+		margin-right: 0;
+
+		&:first-child {
+			border-top-right-radius: 0;
+			border-bottom-right-radius: 0;
+		}
+
+		&:last-child {
+			border-top-left-radius: 0;
+			border-bottom-left-radius: 0;
+		}
+
+		+ .ee-btn {
+			margin-left: 0;
+			margin-right: var(--ee-margin-default);
+		}
+	}
+}
```

Comment:
I noticed that in react-material example they keep border:

![image](https://user-images.githubusercontent.com/1477453/82931976-61044c00-9f90-11ea-99af-44802a9b5f7d.png)
***
w/o right side border it looks a bit awkward:

![image](https://user-images.githubusercontent.com/1477453/82932037-7da08400-9f90-11ea-814a-86ee986b7c39.png)

Reply:
currently, there is no gap between those 2 buttons when border color is transparent, the only thing that bothers me a little is the border of the inactive btn:

![Peek 2020-05-26 20-39](https://user-images.githubusercontent.com/1477453/82932460-1fc06c00-9f91-11ea-9c4b-7f6fd1a4209f.gif)

Reply:
hmmm... what if we either:

- ditched the borders and just used a 2px `box-shadow` with no blur? Those button group buttons would then overlap their neighbour when the style was applied

- ditch the `box-shadow` and just use borders that are normally the same color as the button background. The the button group buttons would not have any overlap or mismatch in height


if you want, we can just leave this for me to fiddle with since I guarantee I will be really picky about these subtle details.

Reply:
yeah, I got a bit side tracked with the `ButtonGroup`;
created this issues and added your comment there as a reference: https://github.com/eventespresso/event-espresso-core/issues/2874
***
also, added a comment there related to cases when `ButtonGroup` will be wrapping more than 2 buttons, if that'll be the case